src/app/mytools/my-tool/my-tool.component.ts

```diff
@@ -172,10 +196,13 @@ export class MyToolComponent extends MyEntry implements OnInit {
 
   selectEntry(tool: ExtendedDockstoreTool): void {
     if (tool !== null) {
-      this.containersService.getContainer(tool.id, includesValidation).pipe(takeUntil(this.ngUnsubscribe)).subscribe((result) => {
-        this.location.go(this.pageName + '/' + result.tool_path);
-        this.containerService.setTool(result);
-      });
+      this.containersService
+        .getContainer(tool.id, includesValidation)
+        .pipe(takeUntil(this.ngUnsubscribe))
+        .subscribe(result => {
+          this.location.go(this.pageName + '/' + result.tool_path);
```

Comment:
Doesn't `location.go` reload the SPA? (I see it was like this already)

Reply:
doesn't appear to